src/test_l1_plat_power.c

```diff
@@ -750,7 +750,7 @@ void test_l1_plat_power_negative_PLAT_API_GetWakeupSrc (void)
 
```

Comment:
line 305/406/410/413/546/667/681: use UT_ASSERT_EQUAL instead of PLAT_ASSERT_AUTO_TERM_NUMERICAL

line 793: code never reaches to 793 after reset. Better to use UT_ASSERT_EQUAL instead of PLAT_ASSERT_AUTO_TERM_NUMERICAL

Remove the macro definition of PLAT_ASSERT_AUTO_TERM_NUMERICAL

Reply:
Done